cfg_template.yml

```diff
@@ -1,3 +1,7 @@
+prs:
+  # Change this if you use a different name for your default branch (e.g. main)
+  main-branch-name: master
```

Comment:
Should this be commented out by default and have a different value to the hardcoded default, by way of example?